ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/Uniform.hs

```diff
@@ -393,3 +378,20 @@ prop_loeStalling =
         allTips = simpleHash . AF.headHash <$> (btTrunk : suffixes)
 
         suffixes = btbSuffix <$> btBranches
+
+-- | This test sets 'scDowntime', which instructs the scheduler to shut all components down whenever a tick's duration
+-- is greater than 11 seconds, and restarts it while only preserving the immutable DB after advancing the time.
```

Comment:
Aren't we now also preserving the volatile DB?

Reply:
✅